server.js

```diff
@@ -0,0 +1,140 @@
+import bodyParser from 'body-parser';
+import { getUserList ,findUserById } from "./user";
+const express = require("express");
+const app = express();
+const swaggerUi = require("swagger-ui-express");
+const userList = getUserList(); // assume for now this is your database
+const swaggerYaml = require('yamljs');
+const swaggerYamlDocument  = swaggerYaml.load('./swagger.yaml'); 
+
+
+app.use(
+  //'/api-docs',swaggerUi.serve,swaggerUi.setup(swaggerDocument) 
+  '/api-docs',swaggerUi.serve,swaggerUi.setup(swaggerYamlDocument)
+);
+
+app.listen(8000, () => {
+  console.log("server listening on port 8000!");
+  });
+// swaggerDocument = require("./swagger.json");
+
+// // GET Call for all users
```

Comment:
Please remove all this commented code. We'll start from scratch with the APIs soon.